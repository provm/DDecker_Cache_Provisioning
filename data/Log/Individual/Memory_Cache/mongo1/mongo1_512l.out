========== START: Container mongo Memory STATS ==============
cache 62853120
rss 37961728
rss_huge 0
mapped_file 36904960
writeback 0
pgpgin 121088
pgpgout 96475
pgfault 184301
pgmajfault 488
inactive_anon 53248
active_anon 37986304
inactive_file 47796224
active_file 14974976
unevictable 4096
hierarchical_memory_limit 9223372036854771712
total_cache 62853120
total_rss 37961728
total_rss_huge 0
total_mapped_file 36904960
total_writeback 0
total_pgpgin 121088
total_pgpgout 96475
total_pgfault 184301
total_pgmajfault 488
total_inactive_anon 53248
total_active_anon 37986304
total_inactive_file 47796224
total_active_file 14974976
total_unevictable 4096
======== END Container STATS ======
java -cp /home/ubuntu/ycsb_mongo/mongodb-binding/conf:/home/ubuntu/ycsb_mongo/conf:/home/ubuntu/ycsb_mongo/lib/jackson-mapper-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/jackson-core-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/htrace-core4-4.1.0-incubating.jar:/home/ubuntu/ycsb_mongo/lib/core-0.12.0.jar:/home/ubuntu/ycsb_mongo/lib/HdrHistogram-2.1.4.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongo-java-driver-3.0.3.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-core-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-binding-0.12.0.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-classic-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-async-driver-2.0.1.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/slf4j-api-1.6.4.jar com.yahoo.ycsb.Client -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -t
YCSB Client 0.12.0
Command line: -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -t
Loading workload...
Starting test.
Maximum execution time specified as: 600 secs
2017-05-23 23:42:32:990 0 sec: 0 operations; est completion in 0 seconds 
mongo client connection created with mongodb://10.0.3.220:27017/ycsb?w=0
DBWrapper: report latency for each error is false and specific error codes to track for latency are: []
2017-05-23 23:42:42:634 10 sec: 371 operations; 37.1 current ops/sec; est completion in 3 days 2 hours [READ: Count=371, Max=124031, Min=4484, Avg=24392.23, 90=36223, 99=51423, 99.9=124031, 99.99=124031] 
2017-05-23 23:42:52:634 20 sec: 994 operations; 62.3 current ops/sec; est completion in 2 days 7 hours [READ: Count=623, Max=87103, Min=318, Avg=15998.21, 90=26623, 99=43199, 99.9=56319, 99.99=87103] 
2017-05-23 23:43:02:634 30 sec: 1829 operations; 83.5 current ops/sec; est completion in 1 days 21 hours [READ: Count=835, Max=52383, Min=302, Avg=11957.86, 90=20879, 99=38719, 99.9=50399, 99.99=52383] 
2017-05-23 23:43:12:634 40 sec: 2887 operations; 105.8 current ops/sec; est completion in 1 days 14 hours [READ: Count=1058, Max=56543, Min=244, Avg=9433.14, 90=14407, 99=31375, 99.9=45471, 99.99=56543] 
2017-05-23 23:43:22:634 50 sec: 3978 operations; 109.1 current ops/sec; est completion in 1 days 10 hours [READ: Count=1091, Max=47967, Min=224, Avg=9143.64, 90=14199, 99=22719, 99.9=41183, 99.99=47967] 
2017-05-23 23:43:32:634 60 sec: 5095 operations; 111.7 current ops/sec; est completion in 1 days 8 hours [READ: Count=1117, Max=74815, Min=195, Avg=8941.61, 90=13871, 99=36095, 99.9=55743, 99.99=74815] 
2017-05-23 23:43:42:634 70 sec: 6265 operations; 117 current ops/sec; est completion in 1 days 7 hours [READ: Count=1170, Max=54655, Min=179, Avg=8526.6, 90=13423, 99=16071, 99.9=50015, 99.99=54655] 
2017-05-23 23:43:52:634 80 sec: 7489 operations; 122.4 current ops/sec; est completion in 1 days 5 hours [READ: Count=1224, Max=59903, Min=174, Avg=8160.45, 90=12663, 99=16255, 99.9=43487, 99.99=59903] 
2017-05-23 23:44:02:634 90 sec: 8715 operations; 122.6 current ops/sec; est completion in 1 days 4 hours [READ: Count=1226, Max=53919, Min=171, Avg=8142.3, 90=13295, 99=16375, 99.9=43551, 99.99=53919] 
2017-05-23 23:44:12:634 100 sec: 9983 operations; 126.8 current ops/sec; est completion in 1 days 3 hours [READ: Count=1268, Max=42143, Min=168, Avg=7874.27, 90=12863, 99=16559, 99.9=39583, 99.99=42143] 
2017-05-23 23:44:22:634 110 sec: 11264 operations; 128.1 current ops/sec; est completion in 1 days 3 hours [READ: Count=1281, Max=40255, Min=109, Avg=7799.25, 90=12735, 99=15967, 99.9=39007, 99.99=40255] 
2017-05-23 23:44:32:634 120 sec: 12516 operations; 125.2 current ops/sec; est completion in 1 days 2 hours [READ: Count=1252, Max=52095, Min=112, Avg=7979, 90=12951, 99=16183, 99.9=44927, 99.99=52095] 
2017-05-23 23:44:42:634 130 sec: 13810 operations; 129.4 current ops/sec; est completion in 1 days 2 hours [READ: Count=1294, Max=43327, Min=98, Avg=7732.95, 90=13103, 99=17967, 99.9=40063, 99.99=43327] 
2017-05-23 23:44:52:634 140 sec: 15073 operations; 126.3 current ops/sec; est completion in 1 days 1 hours [READ: Count=1263, Max=54847, Min=93, Avg=7909.78, 90=13175, 99=31455, 99.9=51519, 99.99=54847] 
2017-05-23 23:45:02:634 150 sec: 16383 operations; 131 current ops/sec; est completion in 1 days 1 hours [READ: Count=1310, Max=39071, Min=93, Avg=7630.71, 90=12831, 99=18575, 99.9=38751, 99.99=39071] 
2017-05-23 23:45:12:634 160 sec: 17750 operations; 136.7 current ops/sec; est completion in 1 days [READ: Count=1367, Max=39903, Min=90, Avg=7316.17, 90=12983, 99=17855, 99.9=39327, 99.99=39903] 
2017-05-23 23:45:22:634 170 sec: 19127 operations; 137.7 current ops/sec; est completion in 1 days [READ: Count=1377, Max=44159, Min=94, Avg=7258.37, 90=12695, 99=18751, 99.9=40767, 99.99=44159] 
2017-05-23 23:45:32:634 180 sec: 20557 operations; 143 current ops/sec; est completion in 1 days [READ: Count=1430, Max=51295, Min=94, Avg=6992.35, 90=12751, 99=18831, 99.9=50367, 99.99=51295] 
2017-05-23 23:45:42:634 190 sec: 21995 operations; 143.8 current ops/sec; est completion in 23 hours 56 minutes [READ: Count=1438, Max=58623, Min=92, Avg=6946.04, 90=12959, 99=18799, 99.9=49375, 99.99=58623] 
2017-05-23 23:45:52:634 200 sec: 23481 operations; 148.6 current ops/sec; est completion in 23 hours 36 minutes [READ: Count=1486, Max=49567, Min=92, Avg=6729.26, 90=12775, 99=18383, 99.9=39167, 99.99=49567] 
2017-05-23 23:46:02:634 210 sec: 25037 operations; 155.6 current ops/sec; est completion in 23 hours 14 minutes [READ: Count=1556, Max=43711, Min=91, Avg=6426.86, 90=12463, 99=16231, 99.9=36351, 99.99=43711] 
2017-05-23 23:46:12:634 220 sec: 26491 operations; 145.4 current ops/sec; est completion in 23 hours [READ: Count=1454, Max=44479, Min=92, Avg=6876.21, 90=13367, 99=18255, 99.9=40639, 99.99=44479] 
2017-05-23 23:46:22:634 230 sec: 27975 operations; 148.4 current ops/sec; est completion in 22 hours 46 minutes [READ: Count=1484, Max=41567, Min=93, Avg=6726.06, 90=12591, 99=18751, 99.9=40767, 99.99=41567] 
2017-05-23 23:46:32:634 240 sec: 29490 operations; 151.5 current ops/sec; est completion in 22 hours 32 minutes [READ: Count=1515, Max=41279, Min=92, Avg=6606.94, 90=13167, 99=18239, 99.9=36863, 99.99=41279] 
2017-05-23 23:46:42:634 250 sec: 31060 operations; 157 current ops/sec; est completion in 22 hours 17 minutes [READ: Count=1570, Max=42143, Min=93, Avg=6362.3, 90=13127, 99=18831, 99.9=38463, 99.99=42143] 
2017-05-23 23:46:52:634 260 sec: 32680 operations; 162 current ops/sec; est completion in 22 hours 1 minutes [READ: Count=1620, Max=43391, Min=92, Avg=6173.64, 90=12575, 99=18399, 99.9=36191, 99.99=43391] 
2017-05-23 23:47:02:634 270 sec: 34296 operations; 161.6 current ops/sec; est completion in 21 hours 47 minutes [READ: Count=1616, Max=45439, Min=92, Avg=6183.81, 90=12639, 99=19151, 99.9=38879, 99.99=45439] 
2017-05-23 23:47:12:634 280 sec: 35980 operations; 168.4 current ops/sec; est completion in 21 hours 32 minutes [READ: Count=1684, Max=51519, Min=91, Avg=5935.39, 90=12447, 99=19263, 99.9=37855, 99.99=51519] 
2017-05-23 23:47:22:634 290 sec: 37678 operations; 169.8 current ops/sec; est completion in 21 hours 17 minutes [READ: Count=1698, Max=40927, Min=94, Avg=5884.03, 90=12951, 99=18703, 99.9=36479, 99.99=40927] 
2017-05-23 23:47:32:634 300 sec: 39407 operations; 172.9 current ops/sec; est completion in 21 hours 3 minutes [READ: Count=1729, Max=43551, Min=91, Avg=5786.75, 90=12175, 99=19071, 99.9=30863, 99.99=43551] 
2017-05-23 23:47:42:634 310 sec: 41189 operations; 178.2 current ops/sec; est completion in 20 hours 49 minutes [READ: Count=1782, Max=41055, Min=91, Avg=5608.27, 90=12775, 99=21599, 99.9=39743, 99.99=41055] 
2017-05-23 23:47:52:634 320 sec: 42987 operations; 179.8 current ops/sec; est completion in 20 hours 35 minutes [READ: Count=1798, Max=53151, Min=93, Avg=5558.36, 90=12175, 99=19119, 99.9=37151, 99.99=53151] 
2017-05-23 23:48:02:634 330 sec: 44854 operations; 186.7 current ops/sec; est completion in 20 hours 20 minutes [READ: Count=1867, Max=38175, Min=89, Avg=5351.12, 90=12447, 99=19343, 99.9=36543, 99.99=38175] 
2017-05-23 23:48:12:634 340 sec: 46765 operations; 191.1 current ops/sec; est completion in 20 hours 6 minutes [READ: Count=1911, Max=44383, Min=90, Avg=5233.41, 90=12151, 99=19551, 99.9=36703, 99.99=44383] 
2017-05-23 23:48:22:634 350 sec: 48727 operations; 196.2 current ops/sec; est completion in 19 hours 51 minutes [READ: Count=1962, Max=54623, Min=89, Avg=5090.88, 90=12223, 99=19295, 99.9=39359, 99.99=54623] 
2017-05-23 23:48:32:634 360 sec: 50812 operations; 208.5 current ops/sec; est completion in 19 hours 34 minutes [READ: Count=2085, Max=45183, Min=88, Avg=4795.34, 90=11951, 99=18847, 99.9=38943, 99.99=45183] 
2017-05-23 23:48:42:634 370 sec: 52824 operations; 201.2 current ops/sec; est completion in 19 hours 21 minutes [READ: Count=2012, Max=39615, Min=90, Avg=4970, 90=12223, 99=20463, 99.9=39263, 99.99=39615] 
2017-05-23 23:48:52:634 380 sec: 54914 operations; 209 current ops/sec; est completion in 19 hours 7 minutes [READ: Count=2090, Max=45599, Min=88, Avg=4782.68, 90=12287, 99=19663, 99.9=36511, 99.99=45599] 
2017-05-23 23:49:02:634 390 sec: 57047 operations; 213.3 current ops/sec; est completion in 18 hours 52 minutes [READ: Count=2133, Max=66239, Min=89, Avg=4683.03, 90=11967, 99=18959, 99.9=38143, 99.99=66239] 
2017-05-23 23:49:12:634 400 sec: 59110 operations; 206.3 current ops/sec; est completion in 18 hours 41 minutes [READ: Count=2063, Max=38399, Min=89, Avg=4840.81, 90=12127, 99=20959, 99.9=37535, 99.99=38399] 
2017-05-23 23:49:22:634 410 sec: 61197 operations; 208.7 current ops/sec; est completion in 18 hours 29 minutes [READ: Count=2087, Max=42399, Min=89, Avg=4791.29, 90=11959, 99=18975, 99.9=40735, 99.99=42399] 
2017-05-23 23:49:32:634 420 sec: 63370 operations; 217.3 current ops/sec; est completion in 18 hours 17 minutes [READ: Count=2173, Max=46623, Min=90, Avg=4602.67, 90=11999, 99=19999, 99.9=38175, 99.99=46623] 
2017-05-23 23:49:42:634 430 sec: 65493 operations; 212.3 current ops/sec; est completion in 18 hours 7 minutes [READ: Count=2123, Max=47999, Min=91, Avg=4706.02, 90=11975, 99=19279, 99.9=38591, 99.99=47999] 
2017-05-23 23:49:52:634 440 sec: 67670 operations; 217.7 current ops/sec; est completion in 17 hours 56 minutes [READ: Count=2177, Max=44799, Min=90, Avg=4591.3, 90=12111, 99=19743, 99.9=37151, 99.99=44799] 
2017-05-23 23:50:02:634 450 sec: 69733 operations; 206.3 current ops/sec; est completion in 17 hours 48 minutes [READ: Count=2063, Max=43839, Min=89, Avg=4843.63, 90=12007, 99=19055, 99.9=37887, 99.99=43839] 
2017-05-23 23:50:12:634 460 sec: 71830 operations; 209.7 current ops/sec; est completion in 17 hours 39 minutes [READ: Count=2097, Max=63871, Min=90, Avg=4764.93, 90=12031, 99=19071, 99.9=45183, 99.99=63871] 
2017-05-23 23:50:22:634 470 sec: 73967 operations; 213.7 current ops/sec; est completion in 17 hours 31 minutes [READ: Count=2137, Max=51583, Min=89, Avg=4679.57, 90=12055, 99=19023, 99.9=39071, 99.99=51583] 
2017-05-23 23:50:32:634 480 sec: 76159 operations; 219.2 current ops/sec; est completion in 17 hours 22 minutes [READ: Count=2192, Max=43295, Min=90, Avg=4558.95, 90=11631, 99=18895, 99.9=32719, 99.99=43295] 
2017-05-23 23:50:42:634 490 sec: 78251 operations; 209.2 current ops/sec; est completion in 17 hours 15 minutes [READ: Count=2092, Max=43583, Min=89, Avg=4777.31, 90=12039, 99=19295, 99.9=32991, 99.99=43583] 
2017-05-23 23:50:52:634 500 sec: 80344 operations; 209.3 current ops/sec; est completion in 17 hours 8 minutes [READ: Count=2093, Max=66879, Min=88, Avg=4771.1, 90=11975, 99=17983, 99.9=37055, 99.99=66879] 
2017-05-23 23:51:02:634 510 sec: 82434 operations; 209 current ops/sec; est completion in 17 hours 2 minutes [READ: Count=2090, Max=43871, Min=90, Avg=4784.13, 90=12095, 99=20287, 99.9=39967, 99.99=43871] 
2017-05-23 23:51:12:634 520 sec: 84513 operations; 207.9 current ops/sec; est completion in 16 hours 56 minutes [READ: Count=2079, Max=47935, Min=90, Avg=4810.67, 90=12087, 99=19311, 99.9=45695, 99.99=47935] 
2017-05-23 23:51:22:634 530 sec: 86674 operations; 216.1 current ops/sec; est completion in 16 hours 50 minutes [READ: Count=2161, Max=41151, Min=88, Avg=4623.44, 90=12063, 99=20863, 99.9=37951, 99.99=41151] 
2017-05-23 23:51:32:634 540 sec: 88707 operations; 203.3 current ops/sec; est completion in 16 hours 45 minutes [READ: Count=2033, Max=44831, Min=92, Avg=4912.96, 90=12055, 99=20127, 99.9=36831, 99.99=44831] 
2017-05-23 23:51:42:634 550 sec: 90757 operations; 205 current ops/sec; est completion in 16 hours 40 minutes [READ: Count=2050, Max=38239, Min=92, Avg=4878.6, 90=12167, 99=18911, 99.9=35295, 99.99=38239] 
2017-05-23 23:51:52:634 560 sec: 92828 operations; 207.1 current ops/sec; est completion in 16 hours 36 minutes [READ: Count=2071, Max=38047, Min=90, Avg=4825.78, 90=12159, 99=20367, 99.9=36895, 99.99=38047] 
2017-05-23 23:52:02:634 570 sec: 94928 operations; 210 current ops/sec; est completion in 16 hours 31 minutes [READ: Count=2100, Max=48191, Min=91, Avg=4760.87, 90=12023, 99=18767, 99.9=34239, 99.99=48191] 
2017-05-23 23:52:12:634 580 sec: 96989 operations; 206.1 current ops/sec; est completion in 16 hours 27 minutes [READ: Count=2061, Max=37951, Min=90, Avg=4846.74, 90=12015, 99=17871, 99.9=35743, 99.99=37951] 
2017-05-23 23:52:22:634 590 sec: 99074 operations; 208.5 current ops/sec; est completion in 16 hours 22 minutes [READ: Count=2085, Max=40191, Min=90, Avg=4793.91, 90=11863, 99=20399, 99.9=39007, 99.99=40191] 
2017-05-23 23:52:32:634 600 sec: 101260 operations; 218.6 current ops/sec; est completion in 16 hours 17 minutes [READ: Count=2186, Max=35839, Min=89, Avg=4572.24, 90=11807, 99=18783, 99.9=35359, 99.99=35839] 
Maximum time elapsed. Requesting stop for the workload.
Stop requested for workload. Now Joining!
2017-05-23 23:52:32:664 600 sec: 101267 operations; 233.33 current ops/sec; est completion in 16 hours 17 minutes [CLEANUP: Count=1, Max=1150, Min=1150, Avg=1150, 90=1150, 99=1150, 99.9=1150, 99.99=1150] [READ: Count=7, Max=11479, Min=141, Avg=4548.86, 90=8575, 99=11479, 99.9=11479, 99.99=11479] 
[OVERALL], RunTime(ms), 600031.0
[OVERALL], Throughput(ops/sec), 168.76961356996554
[TOTAL_GCS_PS_Scavenge], Count, 169.0
[TOTAL_GC_TIME_PS_Scavenge], Time(ms), 234.0
[TOTAL_GC_TIME_%_PS_Scavenge], Time(%), 0.038997985104102954
[TOTAL_GCS_PS_MarkSweep], Count, 0.0
[TOTAL_GC_TIME_PS_MarkSweep], Time(ms), 0.0
[TOTAL_GC_TIME_%_PS_MarkSweep], Time(%), 0.0
[TOTAL_GCs], Count, 169.0
[TOTAL_GC_TIME], Time(ms), 234.0
[TOTAL_GC_TIME_%], Time(%), 0.038997985104102954
[CLEANUP], Operations, 1.0
[CLEANUP], AverageLatency(us), 1150.0
[CLEANUP], MinLatency(us), 1150.0
[CLEANUP], MaxLatency(us), 1150.0
[CLEANUP], 95thPercentileLatency(us), 1150.0
[CLEANUP], 99thPercentileLatency(us), 1150.0
[READ], Operations, 101267.0
[READ], AverageLatency(us), 5911.726495304492
[READ], MinLatency(us), 88.0
[READ], MaxLatency(us), 124031.0
[READ], 95thPercentileLatency(us), 15007.0
[READ], 99thPercentileLatency(us), 22271.0
[READ], Return=OK, 101267
========== STOP: Container mongo Memory STATS ==============
cache 744271872
rss 255537152
rss_huge 0
mapped_file 8544256
writeback 0
pgpgin 785265
pgpgout 541171
pgfault 242945
pgmajfault 529
inactive_anon 129159168
active_anon 126447616
inactive_file 372039680
active_file 372158464
unevictable 4096
hierarchical_memory_limit 999997440
total_cache 744271872
total_rss 255537152
total_rss_huge 0
total_mapped_file 8544256
total_writeback 0
total_pgpgin 785265
total_pgpgout 541171
total_pgfault 242945
total_pgmajfault 529
total_inactive_anon 129159168
total_active_anon 126447616
total_inactive_file 372039680
total_active_file 372158464
total_unevictable 4096
======== END Container STATS ======
