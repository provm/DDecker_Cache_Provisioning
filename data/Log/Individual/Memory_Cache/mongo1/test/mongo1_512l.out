========== START: Container mongo Memory STATS ==============
cache 62840832
rss 37945344
rss_huge 0
mapped_file 36917248
writeback 0
pgpgin 120825
pgpgout 96219
pgfault 183944
pgmajfault 488
inactive_anon 53248
active_anon 37969920
inactive_file 47767552
active_file 14991360
unevictable 4096
hierarchical_memory_limit 9223372036854771712
total_cache 62840832
total_rss 37945344
total_rss_huge 0
total_mapped_file 36917248
total_writeback 0
total_pgpgin 120825
total_pgpgout 96219
total_pgfault 183944
total_pgmajfault 488
total_inactive_anon 53248
total_active_anon 37969920
total_inactive_file 47767552
total_active_file 14991360
total_unevictable 4096
======== END Container STATS ======
java -cp /home/ubuntu/ycsb_mongo/mongodb-binding/conf:/home/ubuntu/ycsb_mongo/conf:/home/ubuntu/ycsb_mongo/lib/jackson-mapper-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/jackson-core-asl-1.9.4.jar:/home/ubuntu/ycsb_mongo/lib/htrace-core4-4.1.0-incubating.jar:/home/ubuntu/ycsb_mongo/lib/core-0.12.0.jar:/home/ubuntu/ycsb_mongo/lib/HdrHistogram-2.1.4.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongo-java-driver-3.0.3.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-core-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-binding-0.12.0.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/logback-classic-1.1.2.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/mongodb-async-driver-2.0.1.jar:/home/ubuntu/ycsb_mongo/mongodb-binding/lib/slf4j-api-1.6.4.jar com.yahoo.ycsb.Client -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -load
YCSB Client 0.12.0
Command line: -db com.yahoo.ycsb.db.MongoDbClient -s -P /home/ubuntu/ycsb_mongo/workloads/workload1 -p mongodb.url=mongodb://10.0.3.220:27017/ycsb?w=0 -p maxexecutiontime=600 -load
Loading workload...
Starting test.
Maximum execution time specified as: 600 secs
2017-05-21 11:47:25:496 0 sec: 0 operations; est completion in 0 seconds 
mongo client connection created with mongodb://10.0.3.220:27017/ycsb?w=0
DBWrapper: report latency for each error is false and specific error codes to track for latency are: []
2017-05-21 11:47:35:125 10 sec: 3607 operations; 360.7 current ops/sec; est completion in 36 minutes [INSERT: Count=3607, Max=499455, Min=111, Avg=2414.15, 90=490, 99=2991, 99.9=395519, 99.99=499455] 
2017-05-21 11:47:45:125 20 sec: 71524 operations; 6791.7 current ops/sec; est completion in 3 minutes [INSERT: Count=67921, Max=330751, Min=32, Avg=141.91, 90=102, 99=134, 99.9=6331, 99.99=157055] 
2017-05-21 11:47:55:125 30 sec: 209561 operations; 13803.7 current ops/sec; est completion in 1 minutes [INSERT: Count=138038, Max=2547, Min=49, Avg=67.92, 90=98, 99=108, 99.9=136, 99.99=1235] 
2017-05-21 11:48:05:125 40 sec: 379329 operations; 16976.8 current ops/sec; est completion in 45 seconds [INSERT: Count=169768, Max=2671, Min=33, Avg=55, 90=59, 99=70, 99.9=102, 99.99=1211] 
2017-05-21 11:48:15:125 50 sec: 547930 operations; 16860.1 current ops/sec; est completion in 24 seconds [INSERT: Count=168600, Max=3445, Min=49, Avg=55.46, 90=60, 99=91, 99.9=110, 99.99=1301] 
2017-05-21 11:48:25:125 60 sec: 717432 operations; 16950.2 current ops/sec; est completion in 7 seconds [INSERT: Count=169502, Max=2899, Min=49, Avg=55.17, 90=59, 99=72, 99.9=113, 99.99=1285] 
Could not wait until max specified time, TerminatorThread interrupted.
2017-05-21 11:48:29:978 64 sec: 800000 operations; 17013.81 current ops/sec; [CLEANUP: Count=1, Max=984, Min=984, Avg=984, 90=984, 99=984, 99.9=984, 99.99=984] [INSERT: Count=82564, Max=2455, Min=49, Avg=54.86, 90=59, 99=71, 99.9=107, 99.99=1258] 
[OVERALL], RunTime(ms), 64854.0
[OVERALL], Throughput(ops/sec), 12335.39951275172
[TOTAL_GCS_PS_Scavenge], Count, 441.0
[TOTAL_GC_TIME_PS_Scavenge], Time(ms), 482.0
[TOTAL_GC_TIME_%_PS_Scavenge], Time(%), 0.743207820643291
[TOTAL_GCS_PS_MarkSweep], Count, 0.0
[TOTAL_GC_TIME_PS_MarkSweep], Time(ms), 0.0
[TOTAL_GC_TIME_%_PS_MarkSweep], Time(%), 0.0
[TOTAL_GCs], Count, 441.0
[TOTAL_GC_TIME], Time(ms), 482.0
[TOTAL_GC_TIME_%], Time(%), 0.743207820643291
[CLEANUP], Operations, 1.0
[CLEANUP], AverageLatency(us), 984.0
[CLEANUP], MinLatency(us), 984.0
[CLEANUP], MaxLatency(us), 984.0
[CLEANUP], 95thPercentileLatency(us), 984.0
[CLEANUP], 99thPercentileLatency(us), 984.0
[INSERT], Operations, 800000.0
[INSERT], AverageLatency(us), 75.36272375
[INSERT], MinLatency(us), 32.0
[INSERT], MaxLatency(us), 499455.0
[INSERT], 95thPercentileLatency(us), 96.0
[INSERT], 99thPercentileLatency(us), 108.0
[INSERT], Return=OK, 800000
========== STOP: Container mongo Memory STATS ==============
cache 103833600
rss 392167424
rss_huge 0
mapped_file 37576704
writeback 0
pgpgin 219195
pgpgout 98101
pgfault 273210
pgmajfault 500
inactive_anon 45056
active_anon 392192000
inactive_file 87969792
active_file 15790080
unevictable 4096
hierarchical_memory_limit 1073741824
total_cache 103833600
total_rss 392167424
total_rss_huge 0
total_mapped_file 37576704
total_writeback 0
total_pgpgin 219195
total_pgpgout 98101
total_pgfault 273210
total_pgmajfault 500
total_inactive_anon 45056
total_active_anon 392192000
total_inactive_file 87969792
total_active_file 15790080
total_unevictable 4096
======== END Container STATS ======
